src/game.rs

```diff
@@ -95,6 +96,7 @@ impl Games {
                         south: participants[2].player.clone(),
                         west: participants[3].player.clone(),
                         rules: participants[0].rules,
+                        created_at: created_at.clone(),
```

Comment:
If this remained as a reference, this is more idiomatic as `created_at: *created_at,`. When it's not a reference, it can simply be `created_at,`